osu.Game/Skinning/Skin.cs

```diff
@@ -239,5 +246,50 @@ protected virtual void Dispose(bool isDisposing)
         }
 
         #endregion
+
+        public override string ToString() => $"{GetType().ReadableName()}{{ Name: {Name} }}";
```

Comment:
This is gonna be an all-timer nitpick but the whitespace is kinda bothering me here... I'd expect something along the lines of

```diff
diff --git a/osu.Game/Skinning/Skin.cs b/osu.Game/Skinning/Skin.cs
index d979cab72b..ffb7fd2669 100644
--- a/osu.Game/Skinning/Skin.cs
+++ b/osu.Game/Skinning/Skin.cs
@@ -247,7 +247,7 @@ protected virtual void Dispose(bool isDisposing)
 
         #endregion
 
-        public override string ToString() => $"{GetType().ReadableName()}{{ Name: {Name} }}";
+        public override string ToString() => $"{GetType().ReadableName()} {{ Name: {Name} }}";
 
         private static readonly ThreadLocal<int> nested_level = new ThreadLocal<int>(() => 0);
 

```
Was the whitespace on purpose here? I'll even fix myself if it wasn't, just wasn't 100% sure...

Reply:
ah yeah, yours makes more sense